temoa/extensions/monte_carlo/mc_run.py

```
@@ -114,7 +117,8 @@ def row_parser(self, row_number: int, row: str) -> RowData:
         # check length
         if len(tokens) != 6:
             raise ValueError(
-                f'Incorrect number of tokens for row {row_number}.  Did you omit notes / trailing comma for no notes or have a comma in your note?'
+                f'Error parsing line {row_number}.  Did you omit notes / trailing comma for no '
+                'notes or have a comma in your note?'
             )
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)